mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule Telemetry.Metrics.MixProject do
 
   defp deps do
     [
-      {:telemetry, "~> 0.2"},
+      {:telemetry, "~> 0.3.0"},
```

Comment:
I think we can go with `~> 0.3` since this project is pre-1.0 also. Otherwise we will need to do a bunch of releases on every telemetry new release!

Reply:
True, let's go with 0.3 :)